### PR TITLE
Add get_current_scope()  (fixes #952)

### LINKED ISSUE #952: Add top-level getter for current scope
The only way I could find is `slash.context.session.scope_manager.get_current_stack()[-1]` which is not very friendly.

Maybe some top-level method could be added to get the current scope?

### PR DESCRIPTION
**Pull request by ayalash**

### CODE REVIEW

File: doc/changelog.rst [hunk 1]
```diff
@@ -1,5 +1,6 @@
 Changelog
 =========
+* :feature:`952` Support API ``get_current_scope()`` and ``ScopeManager.current_scope``
```
Comment (vmalloc) [inline doc/changelog.rst]: `Added support for getting the currently active scope (``test``, ``module`` or ``session``) through the new ``get_current_scope`` API. ``session.scopes.current_scope`` is also available`

Comment (vmalloc) [reply doc/changelog.rst]: Double-check me on that last one please...

Comment (ayalash) [reply doc/changelog.rst]: The last one should be ``session.scope_manager.current_scope``. I'll fix it.

**Comment by ayalash** [comment]
@vmalloc Fixed. Please review.

**Pull request merged**